### DISCUSSION TITLE
video overview of how grizzly works

### DISCUSSION THREAD

**Discussion by calebAtIspot** [grafana/grizzly#533] [Ideas]
I read through the docs but I'm still not completely clear on how grizzly works in practice and the benefits it brings to the table. It would be nice if there was a video walking one through it. No specific questions, I'm just not very confident in my understanding after reading through the docs and feel like a video would help.

**Comment by malcolmholmes** [comment]
It isn't all use-cases for Grizzly, but there's a video here:

https://grafana.com/blog/2024/10/29/edit-your-git-based-grafana-dashboards-locally/
